asv_bench/benchmarks/arithmetic.py

```diff
@@ -466,7 +466,7 @@ def setup(self, offset):
         self.rng = rng
 
     def time_apply_index(self, offset):
-        offset.apply_index(self.rng)
```

Comment:
@jbrockmendel I guess BaseOffset is not used by the non-tick offsets.

Reply:
and we don't have any tests for this public-ish method

Reply:
we should just privatize

Reply:
same for apply

Reply:
agree we *should*, but need to deprecate unfortunately......an you open an issue?